src/mongo/db/catalog/catalog_raii.cpp

```diff
@@ -29,6 +29,7 @@
 #include "mongo/platform/basic.h"
 
 #include "mongo/db/catalog/catalog_raii.h"
+#include "mongo/db/curop.h"
```

Comment:
I don't know why the blank line below this was present before, but it was probably a mistake. If you delete the line below this one and sort the block of include directives, it will be more consistent with other places in our codebase.